ssl/quic/quic_local.h

```diff
@@ -205,6 +205,12 @@ struct quic_conn_st {
     unsigned int                    addressed_mode_w        : 1;
     unsigned int                    addressed_mode_r        : 1;
 
+    /* Flag to indicate if this connection has been accepted */
+    unsigned int                    accepted                : 1;
```

Comment:
Don't think this is needed any more (and what's the difference between `pending` anyway?